Source/RapyutaSimulationPlugins/Private/Core/RRNetworkGameMode.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020-2022 Rapyuta Robotics Co., Ltd.
+
+#include "Core/RRNetworkGameMode.h"
+
+// RapyutaSimulationPlugins
+#include "Core/RRNetworkGameState.h"
+#include "Core/RRNetworkPlayerController.h"
+#include "Tools/RRROS2SimulationStateClient.h"
+
+ARRNetworkGameMode::ARRNetworkGameMode()
+{
+    GameStateClass = ARRNetworkGameState::StaticClass();
+    PlayerControllerClass = ARRNetworkPlayerController::StaticClass();
+}
+
+void ARRNetworkGameMode::PostLogin(APlayerController* InPlayerController)
+{
+    Super::PostLogin(InPlayerController);
+
+    auto* networkPlayerController = CastChecked<ARRNetworkPlayerController>(InPlayerController);
+#if WITH_EDITOR
+    FString pcName = (NetworkClientControllerList.Num() == 0)
+                       ? URRCoreUtils::PIXEL_STREAMER_PLAYER_NAME
+                       : FString::Printf(TEXT("NetworkPC%d"), NetworkClientControllerList.Num());
+    // pc's server name
+    networkPlayerController->SetName(pcName);
+    // pc's client local name
+    networkPlayerController->Rename(*pcName);
```

Comment:
Do we need rename? SetName do not set both client and server?

Reply:
Actually the SetName and Rename functions each rename the player and the player controller's name in the editor, mainly so that we can see it easier in the editor and identify them